corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -248,6 +249,35 @@ def _get_forms_for_user(domain, user_id, state, ids_only=False):
             return [form.form_id for form in forms]
         return forms
 
+    @staticmethod
+    def get_all_forms_received_since(received_on_since=None, chunk_size=500):
+        start_from = received_on_since or datetime.min
+        # todo: this will greedily query the same cases multiple times in a sharded
+        # setup. We should make it smarter
```

Comment:
yeah, definitely will want to eventually come back to this. right now that's what it will do but in the future we could do something smarter where since we know we're getting everything we can just keep a date pointer for each shard and stitch the results together accordingly without going back to the db. seemed a little bit complicated so i punted it for now